src/App.jsx

```diff
@@ -294,8 +294,15 @@ function App() {
           </div>
         )}
       </div>
-      <Dialog open={openNewUpload} onClose={() => setOpenNewUpload(false)}>
-        <DialogTitle>New Upload</DialogTitle>
+      <Dialog style={{borderRadius: "100px"}} open={openNewUpload} onClose={() => setOpenNewUpload(false)}>
+        <div style={{padding: "20px", borderRadius: "10%"}}>
+        <center><img
```

Comment:
don't use deprecated tags like centered and update it with modern css alignment approach. You might refer to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center